python/Scripts/baketextures.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+'''
+Generate the baked mtlx file and baked textures programmatically.
+'''
+
+import sys, os, string
+import glfw
+import struct
+import MaterialX as mx
+from MaterialX import PyMaterialXRender as r
+from MaterialX import PyMaterialXRenderGlsl as rg
+from MaterialX import PyMaterialXGenShader as gs
```

Comment:
You should keep these expanded to improve readability. `mx` might be ok, but even there `mtlx` would probably be better.

`from MaterialX import PyMaterialXRenderGlsl as mtlx_render`

Reply:
That's a good suggestion, and we might consider using "mx_render", "mx_render_glsl", and "mx_gen_shader" for the local names in this script.

Reply:
You exactly read my mind! Thanks all :)